README.md

```diff
@@ -77,3 +80,33 @@ Sample Record:
 * The zendesk API has a rate limit of 10 requests per minute. If rate limit is exceeded, zendesk sends 429 status code with Cool off duration in `Retry-After` header.
 We use this duration to skip hitting the zendesk APIs repeatedly.
 * Currently, the connector only supports ticket data fetching. Other type of data fetching will be part of subsequent phases.
+
+
+## Destination Connector
+The destination connector receives the records from the conduit as individual record object and store it to the buffer as an array of records. Once the maxBufferSize is reached it will push the tickets to zendesk using [bulk import api](https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_import/#ticket-bulk-import). Configuration for zendesk destination api includes `zendesk.domain`, `zendesk.userName`,`zendesk.apiToken`. Once the zendesk client is initialized, it will wait for the buffer to be filled, before writing the data to zendesk destination account.
```

Comment:
Conduit records can have "raw" payloads and structured payloads. Some connectors accept only one or the other, so we should make it clear here which type of records is this ZenDesk destination able to write (that might limit its usage to only certain sources).